cmd/bb_replicator/BUILD.bazel

```diff
@@ -22,18 +23,43 @@ go_library(
     ],
 )
 
-go_binary(
-    name = "bb_replicator",
-    embed = [":bb_replicator_lib"],
-    pure = "on",
-    visibility = ["//visibility:public"],
-)
+[
+    go_binary(
+        name = "bb_replicator-{}".format(goarch),
+        embed = [":bb_replicator_lib"],
+        pure = "on",
+        goarch = goarch,
+        goos = "linux",
+        visibility = ["//visibility:public"],
+    )
+    for goarch in ["amd64", "arm64"]
+]
 
-go_image(
+[
+    pkg_tar(
+        name = "tar-{}".format(goarch),
+        srcs = ["bb_replicator-{}".format(goarch)],
+        package_dir = "/app/cmd/bb_storage",
```

Comment:
Out of curiosity, why do we want to retain this path prefix?

Reply:
I only did this to minimize the delta with the images you publish today. That's the path I see the binaries currently reside within the image.

Reply:
Feel free to remove it. As long as the resulting image is functionally equivalent, it's fine.